Documentation/5.4.1/README.rst

```diff
@@ -488,6 +488,7 @@ This example shows how to create a deployment with one Controller node and two C
 
     **For multi-role AVRS deployment** environment file can be found at : `compute-avrs-mutlirole-environment.yaml <../../nuage-tripleo-heat-templates/environments/compute-avrs-mutlirole-environment.yaml>`_ file. See the sample in the `Sample Templates`_ section.
 
+    **Please notice these are sample templates and parameter values can be customized depending on the use case. Please contact Nuage for the recommended values for these parameters**.
```

Comment:
Can we move the Note to line 486 before single/multi-role options? other than that LGTM